README.rst

```diff
@@ -38,74 +38,149 @@ This repository uses `git-flow <https://github.com/nvie/gitflow>`_ conventions;
 contains the most recent release, and work in progress will be on the **develop** branch.
 Pull requests should be made against **develop**.
 
-
-Development instructions
-------------------------
-
-Bare-metal
-~~~~~~~~~~
-
-Initial setup and installation:
-
-- Recommended: create and activate a python 3.9 virtualenv::
-
-    virtualenv cdhweb -p python3.9
-    source cdhweb/bin/activate
-
-- Use pip to install required python dependencies.  To install production
-  dependencies only::
-
-    pip install -r requirements.txt
-
-  To install all development requirements::
-
-    pip install -r requirements/dev.txt
-
-- Install django-compressor dependencies::
-
-    npm install
-
-- Copy sample local settings and configure for your environment::
-
-    cp cdhweb/settings/local_settings.py.sample cdhweb/settings/local_settings.py
-
-You must add a ``SECRET_KEY`` value in your local settings.
-
-- Download licensed fonts and install locally under /sitemedia/fonts/
-
-- Install OpenCV dependencies (if necessary) for `wagtail image feature detection <https://docs.wagtail.io/en/stable/advanced_topics/images/feature_detection.html>`_
-
-Docker (Springload)
+-----------------
+Development Setup
+-----------------
+
+Choose one of the following setup options:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Option 1: Local Development ("bare-metal")
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This option runs the application directly on your local machine.
+
+**Setup Steps:**
+
+1. **Set up Python environment:**
+   .. code-block:: bash
+      # Install Python 3.11 (if not already installed)
+      brew install python@3.11  # macOS
+      
+      # Create and activate virtual environment
+      virtualenv cdhweb -p python3.11
+      source cdhweb/bin/activate
```

Comment:
I prefer to use pyenv to manage python versions; pyenv by default will use the `.python-version` file in the main directory, which is currently 3.12.  This file is also used by GitHub Actions to install the correct python version.

I also prefer the builtin `python -m venv` for creating virtualenvs. 

I don't think we need to require everyone to use the same tools, but following the python version seems important since it might cause problems in some cases.